fix typos in javascript documentation (#1063)
Codecademy/docs: content/javascript/concepts/sort/sort.md

---
Title: 'Sort'

> Description: 'In JavaScript, the .sort() method of an array sorts the contents of an array and returns the sorted array. This sorting is done in place and affects the original array. No copy is made. The default sort is in ascending string order. The .sort() method allows the passing of a comparison function to change the ordering of the sort. javascript // Perform the default sort somearray.sort() // Perform the sort using an arrow function for comparisons somearray.sort((A, B) => { ... } )'

Reviewer: ```suggestion
Description: 'Sorts the contents of an array and returns the sorted array. This sorting is done in place and affects the original array.'
```

Author: for me no problem to cut description, so if you say ok, I cut it.

Reviewer: @garanews please do.

## Syntax

> ```javascript

Reviewer 2: ```suggestion
```pseudo
```

// Perform the default sort

> somearray.sort()

Reviewer 2: ```suggestion
someArray.sort()
```

Let's make the same edit for the other lines in this snippet.
